In [1]:
import numpy as np
import pandas as pd
import scipy.stats as stats

step 1: import modules

step 2: create an array values

In [2]:
data = np.array([6,7,7,12,13,13,15,16,19,22])

step 3: calculate the z-scores for each value in the array.

In [3]:
stats.zscore(data)

array([-1.39443338, -1.19522861, -1.19522861, -0.19920477,  0.        ,
        0.        ,  0.39840954,  0.5976143 ,  1.19522861,  1.79284291])

Each z-score tells us how many standard deviation away an individual value is from the mean.
For example: The first value of"6" in the array is 1.394 standard deviation below the mean. The fifth value of"13" in the array is 0 satndard deviations away from the mean
